src/ducks/authentication/MobileRouter.jsx

```diff
@@ -64,6 +86,7 @@ const withAuth = Wrapped => (props, { store }) => {
       }
     }
 
+    cozyClient.login()
```

Comment:
We do not pass the `token` to the login function ?

Reply:
Actually we already retrieve and set the right token from the app's persisted state in https://github.com/cozy/cozy-banks/blob/master/src/ducks/client/mobile.js#L17-L20

So for me it seems OK.

Reply:
If it's already done there, shouldn't we do the login() method there instead of in the router ?

Reply:
`getClient` is executed only when we start the app. We want to execute `login` when the user authenticates. It may occurs on app start, but also during the life of the app, for example if the user logout then login on another instance. That's why the `login` must be in the `onAuthentication` function inside the `MobileRouter` (it is executed whenever the user authenticates, whereas the client is created only on app start to be provided to the `CozyProvider`).